packages/contracts/deploy-config/mainnet.json

```diff
@@ -44,7 +44,7 @@
   "timeLockMinDelaySeconds": 2592000,
   "l2GovernorVotingPeriodBlocks": 129600,
   "l2TimeLockMinDelaySeconds": 604800,
-  "zkVerifierHashScalar": "0x163b85414bb5a9c97d8249f96b9c4ffd8db3e8081133101a14fc8546b704e2b",
-  "zkVerifierM56Px": "0x2aa452092865578d9b68a69f053df3db7a563b7cc7d17c9cce524632ccb30d85",
-  "zkVerifierM56Py": "0xafcccd2c2ebc8ffb530403bb07d4e1bcfc1aa910d4dd08d7cd81fcafef5e556"
+  "zkVerifierHashScalar": "0x2FC715229F110C0240EEE5BDE4C9A1AD3BCD8F344A7450BB54D83BF2AB204F57",
```

Comment:
Did you update mainnet config in advance for v1.4.3?

Reply:
Yes, I did. After QA in Sepolia, I'll deploy the new version of contract and add artifact of it.